test/jasmine/assets/mouse_event.js

```diff
@@ -1,6 +1,11 @@
-module.exports = function(type, x, y, opts) {
+module.exports = function(type, x, y) {
+    var options = {
+        bubbles: true,
+        clientX: x,
+        clientY: y
```

Comment:
Ah makes sense. I was dealing with the opposite problem when working with pie charts - I needed to click on the svg elements and not the other layers.